public/css/nitroxLauncher.css

```diff
@@ -421,23 +343,29 @@ main {
 }
 
 ::-webkit-scrollbar-track {
-    background: none;
+    background: var(--scroll-track)!important;
+    border-radius: 20px;
     /* color of the tracking area */
     border: 4px solid var(--bg-on-dark);
     /* creates padding around scroll thumb */
 }
 
 ::-webkit-scrollbar-thumb {
-    background-color: var(--black-5);
+    background-color: var(--scroll-thumb);
     /* color of the scroll thumb */
     border-radius: 20px;
     /* roundness of the scroll thumb */
-    border: 4px solid var(--bg-on-dark);
+    border: 4px solid transparent;
+    background-clip: padding-box;
     /* creates padding around scroll thumb */
 }
 
 ::-webkit-scrollbar-thumb:window-inactive {
-    background: var(--black-375);
+    opacity: .8;
+}
+
+::selection {
+    background-color: rgba(48, 160, 255, .5);
```

Comment:
For now I'd suggest keeping the native selection colour. Also we get to keep the system inactive/unfocused state that can vary depending on the platform